### PR TITLE
Sr cli roles renamed

### PR DESCRIPTION
**Pull request by bredamc**
Closes # 

### Verification Steps

1. Do x
2. Do y

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### Checklist

- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

**Review by wtrocki — APPROVED** [review]

File: docs/commands/rhoas_service-registry_role_add.md [hunk 1]
```diff
@@ -22,7 +28,7 @@ rhoas service-registry role add --role=admin --username=joedough
 
 ```
       --instance-id string       ID of the Service Registry instance to be used (by default, uses the currently selected instance)
-      --role string              Role to apply
+      --role admin               Role to apply: admin, `manager`, or `viewer`
```
Comment (wtrocki) [inline docs/commands/rhoas_service-registry_role_add.md:31]: missing \` for admin

Comment (wtrocki) [reply docs/commands/rhoas_service-registry_role_add.md:31]: I think this is engine problem and we can ignore it.

Other chars like " ' _ are safer in this space

Comment (craicoverflow) [inline docs/commands/rhoas_service-registry_role_add.md:31]: That looks incorrect - `string` is supposed to indicate the data type, but it has been changed to admin.

```suggestion
      --role string               Role to apply: admin, `manager`, or `viewer`
```

Comment (wtrocki) [reply docs/commands/rhoas_service-registry_role_add.md:31]: this will also break ascidocs lets not use /`

Comment (bredamc) [reply docs/commands/rhoas_service-registry_role_add.md]: Seems that the backticks caused the change from string to admin -- fixed now.

File: docs/commands/rhoas_service-registry_role_add.md [hunk 2]
```diff
@@ -4,7 +4,13 @@ Add or update principal role
...
 ### Synopsis
 
-Add or update role for user or service account
+
+Add or update one of the following roles for a user or service account:
+
+* viewer (provides read access)
+* manager (provides read and write access)
+* admin (enables admin along with read and write access)
+
```
Comment (smccarthy-ie) [inline docs/commands/rhoas_service-registry_role_add.md:14]: I stumbled a bit reading the last line on admin access. You could use "provides admin access and read and write access), or trim it back to something like the following?

* viewer (read access)
* manager (read/write access)
* admin (admin and read/write access)

Comment (bredamc) [reply docs/commands/rhoas_service-registry_role_add.md:14]: @smccarthy-ie Thanks Steve -- updated now :)

**Review by smccarthy-ie — APPROVED** [review]
LGTM, thanks!

**Pull request merged**